### PR TITLE
Attempt to clarify units in docs

### LINKED ISSUE #1739: channelReserve unit unclear from documentation
Expected: API doc clearly states whether `channelReserve` is sat or msat.

Current state: API doc has no such informtion. Quick look at the code suggests that it's most likely in sats. This seems to make sense looking at the data.

Steps to reproduce:

1. open https://acinq.github.io/eclair/#channels-2
2. look at channel reserve

Version and environment: Irrelevant, I believe

### PR DESCRIPTION
**Pull request by Kixunil**
This is a draft of specifying units of fields that are not obvious.

Closes #1739

I inferred the units from context, so please double-check if it's correct. Also please LMK what unit to specify for `htlcMinimum`.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -423,6 +425,16 @@ curl -u :<eclair_api_password> -X POST "http://localhost:8080/channels"
 eclair-cli channels
 ```
 
+The units of returned fields that are not obvious from their names:
+
+field          | unit
+---------------|-----
+dustLimit      | sats
+channelReserve | sats
+htlcMinimum    | ???
```
Comment (t-bast) [inline source/index.html.md:434]: This one is `msats`

File: source/index.html.md [hunk 2]
```diff
@@ -423,6 +425,16 @@ curl -u :<eclair_api_password> -X POST "http://localhost:8080/channels"
...
+toSelfDelay    | blocks
+feeratePerKw   | sats
```
Comment (t-bast) [inline source/index.html.md:436]: This is `sat/kw` to be more precise (bitcoin feerate units can be very confusing - but it's getting better). This also applies to line 653.

File: source/index.html.md [hunk 3]
```diff
@@ -359,6 +359,8 @@ eclair-cli updaterelayfee \
 }
 ```
 
+Returned `feeBase` is in msat.
```
Comment (t-bast) [inline source/index.html.md:362]: Not sure this is necessary, a couple lines below it's obvious that it's `msat`.

**Review by t-bast — COMMENTED** [review]
Looks good, thanks!

File: source/index.html.md (hunk 2 above)
Comment (Kixunil) [reply source/index.html.md:436]: Yeah, it's in name so I feared it'd make it look like sat/kw^2 but maybe it's too strange for anyone to seriously think about it?

File: source/index.html.md (hunk 3 above)
Comment (Kixunil) [reply source/index.html.md:362]: True, shouldn't hurt though, right?

File: source/index.html.md (hunk 2 above)
Comment (t-bast) [reply source/index.html.md:436]: Right, that makes sense, let's keep it like that.

File: source/index.html.md (hunk 3 above)
Comment (t-bast) [reply source/index.html.md:362]: Agreed, let's keep it if you think it's useful.

File: source/index.html.md [hunk 4]
```diff
@@ -630,6 +642,16 @@ curl -u :<eclair_api_password> -X POST -F channelId=<channel>  "http://localhost
 eclair-cli channel --channelId=<channel>
 ```
 
+The units of returned fields that are not obvious from their names:
+
+field          | unit
+---------------|-----
+dustLimit      | sats
+channelReserve | sats
+htlcMinimum    | ???
```
Comment (t-bast) [inline source/index.html.md:651]: You've missed that one, then we should be good to go.

**Comment by Kixunil** [comment]
OK, I've fixed the other `???` and squashed.

**Review by t-bast — APPROVED** [review]

**Pull request merged**